src/nidmm/examples/nidmm_measurement_grpc.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/python
+
+import argparse
+import grpc
+import nidmm
+import sys
+
+
+def example(resource_name, option_string, function, range, digits, address, port):
+    session_name = ''  # user-specified name; empty string means use a new, unnamed session
+
+    # Connect to the grpc server
+    channel = grpc.insecure_channel(f'{address}:{port}')
+    session_options = nidmm.GrpcSessionOptions(channel, session_name, nidmm.SessionInitializationBehavior.AUTO)
```

Comment:
@danestull it might be better to remove the final parameter in these examples. Once [the options-key PR](https://github.com/ni/nimi-python/pull/1855) by @DavidCurtiss goes in, assuming his changes are accepted as-is, the behavior becomes a named parameter. Since it's the default anyway, it would keep the example simple, which is good.

Reply:
Went ahead and removed these from the examples and tested and they all still work appropriately using the default behavior.